embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/LogSinkImpl.kt

```diff
@@ -5,12 +5,14 @@ import io.opentelemetry.sdk.logs.data.LogRecordData
 
 internal class LogSinkImpl : LogSink {
     private val storedLogs: MutableList<EmbraceLogRecordData> = mutableListOf()
+    private val onLogsStored: (() -> Unit)? = null
 
     override fun storeLogs(logs: List<LogRecordData>): CompletableResultCode {
         try {
             synchronized(storedLogs) {
                 storedLogs += logs.map { EmbraceLogRecordData(logRecordData = it) }
             }
+            onLogsStored?.let { it() }
```

Comment:
nit: this could be clearer to read:
```suggestion
            onLogsStored?.invoke()
```